src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -83,6 +84,15 @@ class FileColumnProperties {
     this.setDistinctValues(stats.getDistinctValues());
   }
 
+  @JsonProperty("columnId")
```

Comment:
Should this be `columnOrdinal`?

Reply:
I thought about this but then server side fails to deserialise with unknown field.
We can still detect if it is sent on server side by making it nullable.

Reply:
yes but it looks to me misleading tbh